ndbench-core/src/main/java/com/netflix/ndbench/core/config/NdbenchConfigListener.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2017 Netflix, Inc.  All rights reserved.
```

Comment:
Netflix Gradle wrapper has a flag `licenseFormat` to add Apache License to all files instead of doing it manually.